### PR TITLE
[Google Fonts] Generate urls from proto fetched from fonts.google.com

### PR DESCRIPTION
**Pull request by johnsonmh**
This PR fetches a proto from fonts.google.com and uses it to generate URLs that are stable. 

Closes https://github.com/material-foundation/google-fonts-flutter/issues/26

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+Contributing to the google_fonts package.
+=========================================
+
+If you're interested in contributing to the `google_fonts` package, thank you! We love receiving
```
Comment (clocksmith) [inline CONTRIBUTING.md]: "Thank you for your interest in contributing to the `google_fonts` package!"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ 1. Write a unit test for your change, if possible, in one of the files in `test/`.
+ 1. Update the `CHANGELOG.md` with a new version number, and a description of the change you're 
+ making.
+ 1. Update the `version:` in the `pubspec.yaml` file to your new version.
+ 
+### Review
+ 1. Make sure all the existing tests are passing by using the following command (from the root of 
+ the project): `$ flutter test test/`.
+ 1. Make sure the repo is formatted using `$ flutter format .`.
+ 1. Create a PR merging your branch from your fork into `google-fonts-flutter/master`.
```
Comment (clocksmith) [inline CONTRIBUTING.md]: Create a PR to merge the branch on your fork into...

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+ 1. Add `johnsonmh` and `clocksmith` as reviewers on the PR. We will take a look and add any 
+ comments. When the PR looks ready, we will merge it and update the package on
```
Comment (clocksmith) [inline CONTRIBUTING.md]: When the PR is ready to be merged, we will...

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+ comments. When the PR looks ready, we will merge it and update the package on 
+ [pub.dev](https://pub.dev/packages/google_fonts)! 
+ 
+Updating the fonts
+------------------
+
+If you notice fonts that are on [fonts.google.com](https://fonts.google.com) that do not appear in 
+this package, it means that the generator needs to be run.
+
+Running the generator is easy, and we would be happy to accept PRs with new fonts.
```
Comment (clocksmith) [inline CONTRIBUTING.md]: and we are happy to accept...

File: CONTRIBUTING.md (hunk 1 above)
Comment (johnsonmh) [reply CONTRIBUTING.md]: Done.

File: CONTRIBUTING.md (hunk 2 above)
Comment (johnsonmh) [reply CONTRIBUTING.md]: Done.

File: CONTRIBUTING.md (hunk 3 above)
Comment (johnsonmh) [reply CONTRIBUTING.md]: Done.

File: CONTRIBUTING.md (hunk 4 above)
Comment (johnsonmh) [reply CONTRIBUTING.md]: Done.

**Pull request merged**